templates/incl/editor-and-output.html

```diff
@@ -196,46 +196,35 @@
-                  <div class="relative">
+          <div class="flex justify-between" id="pagination_container">
+            <button class="green-btn previous-tab ltr:ml-1 rtl:mr-1" 
+              href="#"
+              _="on load or click from .tab
+                set :els to getPreviousAndNext()
+                if :els[0]
+                  set *display to 'block'
+                  set the *justify-content of #pagination_container to 'space-between' 
+                  set prevContent to :els[0].innerHTML.trim()
+                else
+                  set the *justify-content of #pagination_container to 'flex-end' 
+                  set *display to 'none'
+                end
+                put '&LongLeftArrow; ' + prevContent into my innerHTML
+                on click call :els[0].click()"></button>
 
-                    <button class="green-btn" disabled>{{_('regress_button')|replace("{level}", prev_level)}}</button>
-
-                    <div class='text-center tooltip opacity-0 hover:opacity-100 duration-300 absolute inset-0 z-10 flex justify-center items-top'>
-                      <span class='tooltiptext text-xs font-semibold rounded shadow-lg p-1 bg-gray-100 -mt-8'> {{_('disabled_button_locked')}} </span>
-                      </span>
-                    </div>
-                  </div>
-            {% endif %}
-            {% if next_level and (not customizations or next_level in customizations['available_levels']) %}
-                <a class="green-btn ltr:ml-1 rtl:mr-1" id="next_level_button" href="{{ hedy_link(next_level, 1) }}">{{_('advance_button')|replace("{level}", next_level)}}</a>
-            {% elif next_level and not (not customizations or next_level in customizations['available_levels']) %}
-                <div class="relative">
-                    <button class="green-btn ltr:ml-1 rtl:mr-1" id="next_level_button" disabled>{{_('advance_button')|replace("{level}", next_level)}}</button>
-                    <div class="absolute w-80 z-10 opacity-0 hover:opacity-100 duration-300 top-0 ltr:right-0 rtl:left-0">
-                      <div class="text-red-500 text-center font-bold bg-gray-100 px-2 py-1 rounded-lg border-4 border-yellow-600 text-sm">
-                          {{_('disabled_button_quiz') if customizations["below_threshold"] else _('disabled_button_locked')}}
-                       </div>
-                    </div>
-                </div>
-            {% endif %}
+            <button class="green-btn next-tab ltr:ml-1 rtl:mr-1" 
+              href="#"
+              _="on load or click from .tab
+                set :els to getPreviousAndNext()
+                if :els[1]
+                  set *display to 'block'
+                  set nextContent to :els[1].innerHTML.trim()
+                else
+                  set *display to 'none'
+                end
+                put nextContent + ' &LongRightArrow;' into my innerHTML
```

Comment:
Can you change this `&LongRightArrow` for `&LongLeftArrow` in case we are in a RTL language, like Arab?

Reply:
Good one! Will add it